docs/topics/compatibility-guides/compatibility-guide-22.md

```diff
@@ -23,6 +23,22 @@ perspective (for example, from Java) is out of the scope of this document.
 
 ## Language
 
+### Remove language versions 1.6 and 1.7
+
+> **Issue**: [KT-71793](https://youtrack.jetbrains.com/issue/KT-71793)
+>
+> **Component**: Core language
```

Comment:
Maybe it makes sense to use something like "The compiler" / "Compiler component" as it's not really related to the language?